tests/test_benchmark.py

```diff
@@ -280,7 +400,199 @@ def test_hif2a():
     benchmark_hif2a(verbose=True, num_batches=2, steps_per_batch=100)
 
 
+def test_ixn_group_potential(hi2fa_test_frames):
+    _, sys_params, frames, boxes, ligand_idxs = hi2fa_test_frames
+    lamb = 0.0
+    num_atoms = frames.shape[1]
+    nonbonded_params = sys_params[3]
+    beta = 1 / (constants.BOLTZ * 300)
+    cutoff = 1.2
+
+    precisions = [np.float32, np.float64]
+
+    potential = NonbondedInteractionGroup(
+        ligand_idxs,
+        np.zeros(num_atoms, dtype=np.int32),
+        np.zeros(num_atoms, dtype=np.int32),
+        beta,
+        cutoff,
+    )
+    class_name = potential.__class__.__name__
+    for precision in precisions:
+        if precision == np.float32:
+            label = class_name + "_f32"
+        else:
+            label = class_name + "_f64"
+        benchmark_potential_over_coords(
+            label,
+            potential,
+            precision,
+            nonbonded_params[: len(nonbonded_params) // 2],
+            frames,
+            boxes,
+            lamb,
+            verbose=False,
+        )
+        benchmark_potential_over_params(
+            label,
+            potential,
+            precision,
+            np.stack([nonbonded_params[: len(nonbonded_params) // 2]] * len(frames)),
+            frames[0],
+            boxes[0],
+            lamb,
+            verbose=False,
+        )
+
+    potential = NonbondedInteractionGroupInterpolated(
+        ligand_idxs,
+        np.zeros(num_atoms, dtype=np.int32),
+        np.zeros(num_atoms, dtype=np.int32),
+        beta,
+        cutoff,
+    )
+    class_name = potential.__class__.__name__
+    for precision in precisions:
+        if precision == np.float32:
+            label = class_name + "_f32"
+        else:
+            label = class_name + "_f64"
+        benchmark_potential_over_coords(
+            label,
+            potential,
+            precision,
+            nonbonded_params,
+            frames,
+            boxes,
+            lamb,
+            verbose=False,
+        )
+        benchmark_potential_over_params(
+            label,
+            potential,
+            precision,
+            np.stack([nonbonded_params] * len(frames)),
+            frames[0],
+            boxes[0],
+            lamb,
+            verbose=False,
+        )
+
+
+def test_nonbonded_potential(hi2fa_test_frames):
+    pots, sys_params, frames, boxes, _ = hi2fa_test_frames
+
+    nonbonded_interp = pots[3]
+    lamb = 0.0
+    nonbonded_params = sys_params[3]
+
+    precisions = [np.float32, np.float64]
+
+    potential = Nonbonded(
```

Comment:
Should we also test the interpolated variant?

Reply:
It tests the interpolated version here, it isn't constructed since it is already constructed: https://github.com/proteneer/timemachine/blob/feature/add-batch-execute-to-potentials/tests/test_benchmark.py#L527-L551